nightfall-optimist/src/utils/auth.mjs

```diff
@@ -0,0 +1,11 @@
+import config from 'config';
+import crypto from 'crypto';
```

Comment:
Smooth and short implementation w/o passport.js. I'm glad we went for the hashing option. Nice :)